src/clips-specs/rcll-central/goal-production.clp

```diff
@@ -758,6 +758,9 @@
               (mode FORMULATED) (parent ?pa-id&~nil)
               (priority ?p&:(> ?p 0))
         )
+  (goal-meta (goal-id ?goal-id) (assigned-to ?robot))
```

Comment:
This rule needs appropriate priority e.g., `?*SALIENCE-GOAL-EXECUTABLE-CHECK*`, because otherwise it will be ignored on selection.

Reply:
Then you can also simplify this rule and remove the ?or, essentially just updating the rule during executability check